src/wallet/fees.h

```diff
@@ -26,6 +26,18 @@ CAmount GetRequiredFee(unsigned int nTxBytes);
  */
 CAmount GetMinimumFee(unsigned int nTxBytes, const CCoinControl& coin_control, const CTxMemPool& pool, const CBlockPolicyEstimator& estimator, FeeCalculation *feeCalc);
 
+/**
+ * Return the minimum required fee taking into account the
```

Comment:
In commit "Add a GetMinimumFeeRate function which is wrapped by GetMinimumFee" (b68ce5a0e638c47140f350cbc40f9cfc85fb4c37)

Could s/fee/fee rate/